skimage/feature/_canny.py

```diff
@@ -161,137 +296,51 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None,
         low_threshold = 0.1
     elif use_quantiles:
         if not(0.0 <= low_threshold <= 1.0):
-            raise ValueError("Quantile thresholds must be between 0 and 1.")
+            raise ValueError("quantile thresholds must be between 0 and 1.")
```

Comment:
Just my curiosity, why this change?

Reply:
I can't remember why I did this :confused: Will revert :pray: